packages/react/src/select/root/SelectRoot.tsx

```diff
@@ -168,7 +175,7 @@ export function SelectRoot<Value, Multiple extends boolean | undefined = false>(
     value,
     controlRef,
     name,
-    getValue: () => value,
+    getValue: () => fieldStringValue,
```

Comment:
Since we only use `fieldStringValue` inside this callback, does it make sense to compute it lazily? 
Or is it used frequently enough to always compute it in the render?